### DISCUSSION TITLE
GitHub Actions Disabled for My Account - Seeking Assistance

### DISCUSSION THREAD

**Discussion by ghost** [community/community#81536] [Actions]
### Select Topic Area

Question

### Body

Hello GitHub Community,

I am reaching out to seek assistance regarding an issue I'm facing with GitHub Actions. Currently, GitHub Actions is disabled for my account, and I'm encountering challenges in resolving this matter.

Issue Details:

GitHub Username: rakeshrepos
Email Address Associated with the Account: rebbavarapurakesh@gmail.com

I wanna use GitHub Actions for getting my YouTube videos once in a month and its unavailability is affecting my workflow. Despite reviewing my account settings and GitHub Actions documentation, I haven't been able to identify the cause of the issue.

Steps Taken:
Checked account settings for any restrictions.
Reviewed GitHub Actions documentation for troubleshooting.
Verified email notifications for any communication from GitHub.
Screenshot:
![image](https://github.com/community/community/assets/58909876/b168d96c-dbd2-410a-b278-1a0c08082279)

Community Assistance:

If anyone in the community has faced a similar issue or has insights into resolving GitHub Actions being disabled, I would greatly appreciate your guidance. Your experiences and suggestions could be invaluable in helping me overcome this obstacle.

Thank you in advance for your time and assistance. I look forward to any advice or recommendations the community may provide.

**Comment by essefiadam** [comment]
it happens for me too
